offlinemaps/src/main/java/org/akvo/flow/offlinemaps/presentation/list/OfflineAreasListActivity.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019 Stichting Akvo (Akvo Foundation)
+ * Copyright (C) 2018 Stichting Akvo (Akvo Foundation)
```

Comment:
Going back in time? 😄

Reply:
oh my bad, (was trying one of Dan's scripts)